Source/Reporting/Policies/Reporting/Notifications/NotificationProcessor.cs

```diff
@@ -75,16 +75,30 @@ public void Process(TextMessageReceived notification)
 
             if (!isTextMessageFormatValid && !unknownDataCollector)
             {
-                caseReporting.ReportInvalidReport(
+                if (dataCollector.InTraining)
+                {
+                    caseReporting.ReportInvalidTrainingReport(
```

Comment:
Is there something up with the indentation here, or is it only github making it look werid?

Reply:
Yes, the indentation was definitely weird.